Description: Stop using alsa/iatomic.h
 Technically this is an API breakage in libasound 1.1.2,
 but it makes sense to me that this header was removed
 by ALSA upstream.
 .
 bristol was the only user in Debian, and even here the
 code that once used it was already commented out.
Author: Adrian Bunk <bunk@stusta.de>
Bug-Debian: https://bugs.debian.org/834180
Forwarded: no

--- bristol-0.60.11.orig/libbristolaudio/audioEngineJack.c
+++ bristol-0.60.11/libbristolaudio/audioEngineJack.c
@@ -39,7 +39,7 @@
 
 #ifdef _BRISTOL_JACK
 #if (BRISTOL_HAS_ALSA == 1)
-#include <alsa/iatomic.h>
+//#include <alsa/iatomic.h>
 #endif
 
 /*
